backend/src/test/java/gov/cdc/usds/simplereport/service/idverification/LiveExperianServiceTest.java

```diff
@@ -0,0 +1,135 @@
+  private static final String SAMPLE_SESSION_ID = "SAMPLE_SESSION_ID";
```

Comment:
It looks like some sensitive data might have gotten into the samples above (PID_PASSWORD in particular) - can you remove it before submitting?

Reply:
Yeah, this is a good catch.  We should't have any of the demo stuff here even though it can't be used by itself.